tests/utils/stl/test/file_parsing.py

```diff
@@ -58,7 +58,7 @@ class _ParseCtx:
 _TAGS_REGEX = re.compile(r'^(?P<tags>(\*|\w+(,\w+)*))\t+(?P<remainder>.*)$')
 _ENV_VAR_MULTI_ITEM_REGEX = re.compile(r'(?P<name>\w+)="(?P<value>.*?)"')
 _CROSSLIST_REGEX = re.compile(r'^RUNALL_CROSSLIST$')
-_EXPECTED_RESULT_REGEX = re.compile(r'^(?P<prefix>.*) (?P<result>.*?)$')
+_EXPECTED_RESULT_REGEX = re.compile(r'^(?P<prefix>[^ ]*) (?P<result>.*?)$')

```

Comment:
This could be significantly stricter. We expect to have 1 or more characters in the prefix (so instead of `*` we should use `+`), we expect the result to be 1 or more word characters ("FAIL", "SKIPPED", etc.), and now that you're excluding spaces from the prefix, I don't think we need the non-greedy matching.